### PR TITLE
[REVIEW[FEATURE][UWANG-163] - integration get list user

### PR DESCRIPTION
**Pull request by azkifairuz**
**example:** [WIP][BUGFIX][FEATURE][BKA-601] - Security Credentials

#### Description

> Description of features, bugs, improvements or any changes in this MR.

#### Ticket Reference(s)

[UWANG-163](https://bluehabit.atlassian.net/browse/UWANG-163)

##### UI Changes

> Screenshot or video of UI-related changes
gambar

##### Coverage

> Test coverage screenshot.

N/A

#### Note(s)

> Mention other user or extra note unrelated to the MR here.

[UWANG-163]: https://bluehabit.atlassian.net/browse/UWANG-163?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: pages/user-management/daftar-pengguna.vue [hunk 1]
```diff
@@ -0,0 +1,97 @@
+<script setup>
+const listUserStore = useListUser();
+
+onMounted(async () => {
+  await listUserStore.fetchUserWithPagination();
+  console.log("datauser", listUserStore.result);
```
Comment (triandamai) [inline pages/user-management/daftar-pengguna.vue]: hapus

File: pages/user-management/daftar-pengguna.vue [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    value: "dateOfRegister",
+  },
+  {
+    name: "Status",
+    value: "status",
+  },
+];
+
+
+const dataUser = listUserStore.data;
```
Comment (triandamai) [inline pages/user-management/daftar-pengguna.vue]: kenapa di bikin variabel? @azkifairuz

**Review by triandamai — CHANGES_REQUESTED** [review]
repisi

File: pages/user-management/daftar-pengguna.vue (hunk 1 above)
Comment (azkifairuz) [reply pages/user-management/daftar-pengguna.vue]: hapus yg log nya apa keselurhan onMounted?

File: pages/user-management/daftar-pengguna.vue (hunk 2 above)
Comment (azkifairuz) [reply pages/user-management/daftar-pengguna.vue]: karena kan  data yg didapet nnti gini
  status_code: number;
  data:Array<UserCredentialResponse>,
  message: string;
  page: string,
  size: string,
  total_data: number,
  total_page: number
  
  nah  bikin variable yg biar langsung ngambil datanya jdinya nnti gua gselalu ngetik lisUseStore.data terus

File: pages/user-management/daftar-pengguna.vue (hunk 1 above)
Comment (azkifairuz) [reply pages/user-management/daftar-pengguna.vue]: done bang

**Pull request merged**